utilities/db-migration/migrate/migrate_db.go

```diff
@@ -12,41 +12,56 @@ import (
 	db "github.com/redhat-appstudio/managed-gitops/backend-shared/config/db"
 )
 
-func Main(op_type string) {
+func Main(op_type string, call_type string) bool {
```

Comment:
Recommend tweaking the method signature to:
```
func Migrate(opType string, migrationPath string) error {
}
```

So that's:
- Change the function name to Migrate (since it's no longer a main function)
- Inside `Migrate` return an error when an error occurs, rather than returning a bool.
- Change params to go parameter syntax
- Migrate should take the path to the migrations dir, rather than a constant indicating the type. It should be up to functions that are calling Migrate to figure out where the path is.